LICENSE

```diff
@@ -1,10 +1,12 @@
-The Ripple Dev Portal and all its original contents are provided under the MIT License. Additional libraries and resources used by the Ripple Dev Portal are subject to their own licenses, as described below.
+The XRP Ledger Dev Portal and all its original contents are provided under the MIT License. Additional libraries and resources used by the XRP Ledger Dev Portal are subject to their own licenses, as described below.
 
 The MIT License (MIT)
 
-Ripple Dev Portal Copyright (c) 2014, 2015 Ripple Labs, Inc.
-Flatdoc Copyright (c) 2013, 2014, Rico Sta. Cruz.
+XRP Ledger Dev Portal Copyright (c) 2019 XRP Ledger Project
```

Comment:
Interesting, who is the "XRP Ledger Project" and who had the copyright on this documentation since 2014?